docs/pages/careers/design-engineer-x-grid.md

```diff
@@ -0,0 +1,119 @@
+- **Experiment and play**. Great, unexpected features and heisenbug fixes have come from a number of sources — relentlessly methodical processes of elimination, free-flowing team collaboration, inspiration from adjacent libraries and projects, and difficult-to-explain individual strokes of brilliance. Whatever your preferred style is for creating new things that others might not have thought of, you'll find a welcome home on the team.
+- **Take ownership of features from idea/mockup to live deployment**.
+  You'll shape and guide the direction of crucial new features, including new components.
+- **You'll be interacting with customers** regularly, handling inbound support and feature requests (every developer helps with developer requests).
+- **Bridge the gap between design and engineering**.
+  Sometimes, you'll be sketching new features on Figma. On other days, you'll be prototyping them in code. 
+
+### The best parts of this job
+
+- **You'll be at the cutting edge of application development** — working on one of the fastest-growing UI libraries on the market.
+- **You'll be part of an active, open, friendly community** of developers that are excited about building awesome applications.
+- **Your role will be key to making MUI the go-to UI framework** for building applications, websites, and design systems with React.
+
+### The worst parts of this job
+
+- **Shifting context.**
+  You will necessarily have to shift context and dive into a different feature before the current one is done.
+  It may even be in an area of the codebase you're unfamiliar with or don't have a ton of understanding about.
+  It's fun, rewarding work, but it can be very challenging.
+- **We move quickly but don't sacrifice quality**.
+  We ship early, often, and quickly. You may not be initially comfortable with the cadence with which we ship high-quality features and improvements to end-users. By doing so, we sacrifice on solving each problem 100% in exchange for fast feedback. Solving 50-70% of the issue with quality should be enough for any given iteration. Our users quickly tell us when we haven't pushed a solution far enough.
+- **MUI X is a large codebase with complex features**. Understanding parts of the implementation might be tricky, but the learning process is rewarding. You may bang your head against the wall at times, but then write tests to assist future you 😌.
+  The work you will be doing is somewhat unique and idiosyncratic. You probably have not had a similar role before.
+- **In open-source, you're faced with a nonstop stream of bug reports and support requests**. That means you need to develop an intuition for when to ignore something, and when to dig in further.
+
+## About you
+
+### Skills you should have
+
+- 3+ years of relevant web front-end development experience at product-driven companies.
+- Great understanding and taste for product design, including visual design fundamentals and user experience principles.
```

Comment:
```suggestion
### Required

- 3+ years of web front-end development at product-driven companies
- Great understanding and taste for product design
- Experience with design systems
- You're highly empathetic
- Fluent written and spoken English skills

### Nice to have (but not required)

- Experience in open-source
- Experience with MUI products
```

Overall, I think we can improve this list by separating what's required and what's nice to have—it will help us with the scorecard and screening (and should also be a general structure for these pages, too, I think). Here's a suggestion to kickstart it — feel free to add more!